.gitignore

```diff
@@ -16,6 +16,7 @@ TAGS
 /pbmigrate
 /pbworker
 /pbstatuser
+/cmd/pbackend/pbackend
```

Comment:
Unrelated but my IDE keeps adding this file also here and I am lazy to fix all of my run configurations.